frontend/src/store/bs-space/actions.ts

```diff
@@ -0,0 +1,72 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import {
+  AsyncThunkConfig,
+  BSObjectDownloadParamsDto,
+  BSObjectGetRequestParamsDto,
+  BSObjectGetResponseDto,
+} from 'common/types/types';
+import { ActionType } from './common';
+import { NotificationMessage, NotificationTitle } from 'common/enums/enums';
+
+const loadObjects = createAsyncThunk<
+  BSObjectGetResponseDto,
+  {
+    id: string;
+    filter: BSObjectGetRequestParamsDto;
+  },
+  AsyncThunkConfig
+>(ActionType.GET_OBJECTS, async ({ filter, id }, { extra }) => {
+  const { bsApi } = extra;
+
+  return bsApi.loadObjects(filter, id);
+});
+
+const downloadObject = createAsyncThunk<
+  Blob,
+  BSObjectDownloadParamsDto & { filename: string },
+  AsyncThunkConfig
+>(ActionType.DOWNLOAD_OBJECT, async (params, { extra }) => {
```

Comment:
using `getState` find the current object by id and then take a name of this object 👍